deployment/terraform/assets/datasource.yaml

```diff
@@ -0,0 +1,14 @@
+# config file version
+apiVersion: 1
+
+# list of datasources to insert/update depending
+# what's available in the database
+datasources:
+- name: loadtest-source
```

Comment:
Please include here the `scrape time` configuration

Reply:
Did you mean to set that as part of the `jsonData` field here: https://grafana.com/docs/grafana/latest/administration/provisioning/#example-datasource-config-file ? Because there's no direct field called `scrape time`.

Reply:
Yes I think it should go in the jsonData and should be calle something like `scrape_time`. Haven't tried yet

Reply:
Seems to be called `timeInterval`.